AUTHORS

```diff
@@ -1,16 +1,12 @@
 # This is the list of GRANTA MI RecordLists API significant contributors.
```

Comment:
The README uses "PyGranta RecordLists" to refer to the package. It seems more accurate than "GRANTA MI RecordLists API" in this context